### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by swami-raj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ you've configured your node to expose the developer options, e.g.
 ```
 
 #### Mainnet Option
-To test with real bitcoin,  you will need to have a local `bitcoind` node running:
+To test with the  real bitcoin,  you will need to have a local `bitcoind` node running:
```
Comment (vincenzopalazzo) [inline README.md:77]:
```suggestion
To test with the bitcoin,  you will need to have a local `bitcoind` node running:
```

NIT, IMO it doesn't sound good.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ c-lightning is a lightweight, highly customizable and [standard compliant][std]
 [![Documentation Status](https://readthedocs.org/projects/lightning/badge/?version=docs)][docs]
 
 This implementation has been in production use on the Bitcoin mainnet since early 2018, with the launch of the [Blockstream Store][blockstream-store-blog].
-We recommend getting started by experimenting on `testnet` (or `regtest`), but the implementation is considered stable and can be safely used on mainnet.
+We recommend getting started by experimenting on `testnet` (or `regtest`), but this implementation is considered stable and can be safely used on mainnet.
```
Comment (vincenzopalazzo) [inline README.md:28]: why we need to switch from `the` to `this`? IMO, `this` assumes that there are other implementations not stable, but the doc tells us only that the c-lightning can be considered stable.

**Review by vincenzopalazzo — COMMENTED** [review]
Again, not the correct person to review the docs change.

This is only my opinion, I usually run out of grammar.

File: README.md (hunk 1 above)
Comment (cdecker) [reply README.md:77]: Yeah, it's bitcoins the currency, not bitcoin the network / protocol

**Comment by cdecker** [comment]
Sadly #4832 was quicker to get the `the` -> `this` fix in and the change in bitcoin to the bitcoin doesn't sound right.

**Pull request closed without merging**